examples/transmitter/swagger_server/test/conftest.py

```diff
@@ -34,7 +43,7 @@ def new_stream() -> Iterator[Stream]:
     client_id = uuid.uuid4().hex
     stream = Stream(client_id, "https://test-case.popular-app.com")
     yield stream
-    Stream.delete(client_id)
+    stream.delete()
```

Comment:
This is fine to leave in there, but I am almost 100% positive we don't need it, since we destroy the database. It's probably still good to leave in though to keep the intention clear.

Reply:
Ok, then I'll leave it as is